rails-snow-queen/app/views/quotes/new.html.erb

```diff
@@ -16,9 +16,43 @@
             <dd class="text-right"><span id="calculatedArea"></span> sq ft</dd>
             <dt class="p-3">Total: </dt>
             <dd class="text-right">$<span id="totalDue"></span></dd>
-            <input id="quoteSubmit" type="submit" value="Submit request for service" class="btn btn-block btn-warning shadow-lg">
+            <input 
+               id="quoteSubmit" 
+               type="submit" 
+               value="Send me this quote" 
+               class="btn btn-block btn-warning shadow-lg"
+               data-toggle="modal" 
+               data-target="#submitQuoteModal"
+            >
          </dl>
       </form>
+      <div class="modal fade" id="submitQuoteModal" tabindex="-1" role="dialog" aria-labelledby="submitQuoteModalTitle" aria-hidden="true">
+         <div class="modal-dialog modal-dialog-centered" role="document">
+            <div class="modal-content">
+               <div class="modal-header">
+                  <h5 class="modal-title">Service details</h5>
+                  <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                     <span aria-hidden="true">&times;</span>
+                  </button>
+               </div>
+               <div class="modal-body">
+                  <p>Address: <span id="addressModal"></span> </p>
+                  <p>Area: <span id="areaModal"></span> sq ft</p>
+                  <p>Total: $<span id="totalModal"></span> </p>
```

Comment:
Consider using `<dl>` instead of `<p>` tags (for semantic purity)

Reply:
Done